script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -124,22 +129,28 @@ Where's Kohaku and Akiha?"
 [sha:23663d4413037c4855ce3de1e24518a69fa9693a]{
 -- Page 359, Offset 15682.
 -- 秋葉さまは先ほど連絡がありましたので、あと30分ほどでお帰りになられるかと」
-Akiha-sama just contacted me to say that she will return in about 30 minutes."
+Akiha-sama just contacted me to say that she will return in about thirty minutes."
 }
 [sha:5f857e92a54148bbbef42b79983753d75f26bb35]{
 -- Page 360, Offset 15683.
 -- 「そっか。じゃあ30分後に。
-"Gotcha. See you in 30, then.
+"Gotcha. See you in thirty, then.
 }
 [sha:fc3587f7c191f673a9a1c0b2d2272f1dfb544707]{
 -- Page 360, Offset 15684.
 -- ずっと外で待っててくれたんだ、翡翠もちょっとは休んでいてくれ。これは主人としての命令だからな」
-You've been waiting outside this whole time, Hisui, you should get some rest too. And that's an order from your master."
+You've been waiting outside for me this whole time, Hisui; you should get some rest as well.
+ And that's an order from your master."
 }
 [sha:0056781f6ea88d08634652c766b1ebaf4c9e15c4]{
 -- Page 361, Offset 15685.
 -- 「では、お言葉に甘えて失礼します。後ほどお迎えにあがりますので、ごゆっくりお休みください」
-"Very well. I shall accept your kind offer, excuse me. I will come greet you later on, so please rest well."
+// Dithered on whether 'fetch' is too imposing for a while.
+// Under the context of fetching him for dinner or something I think it's reasonable?
+// 'Greet' feels weird (they did that), 'call on' is again like she wants something _from_ him,
```

Comment:
"call on you for dinner" would be perfectly acceptable maid-speak but "fetch you for dinner" isn't far outside of servant parlance imo. No change necessary